templates/zerver/app/markdown_help.html

```diff
@@ -82,6 +82,14 @@
                         <td>#**streamName**</td>
```

Comment:
hm, maybe in a commit on top we should change `streamName` to `stream name`, given that this syntax supports spaces.